### PR TITLE
Feature includes

### PR DESCRIPTION
**Pull request by Exelord**

### CODE REVIEW

File: src/application.ts [hunk 1]
```diff
@@ -28,9 +36,11 @@ export default class Application {
   async executeOperations(ops: Operation[]): Promise<OperationResponse[]> {
     return await this.createTransaction(
       ops.map(async op => {
-        const processor = this.processorFor(op);
-        const result = await processor.execute(op);
-        return this.buildOperationResponse(result);
+        const processor = this.processorFor(op.ref.type);
+        const data = await processor.execute(op);
+        const included = await this.resolveIncludes(op, data);
+
```
Comment (ebryn) [inline src/application.ts]: here is probably where you want to now mutate the resource objects in `data` to "inject" the `relationships` object that provides the links to the resources now in `included`

File: tests/dummy/src/resources/user.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Resource } from "../jsonapi-ts";
+
+export default class User extends Resource {
+  static attributes = {
```
Comment (rtablada) [inline tests/dummy/src/resources/user.ts]: How are we declaring that a User has many posts or whatever relationship exists between these two resources?

Comment (Exelord) [reply tests/dummy/src/resources/user.ts]: We don't have resource relationships yet :/

Comment (rtablada) [reply tests/dummy/src/resources/user.ts]: Even if it's something temporary I think we should look at how we declare relations before we can tackle includes.

Comment (ebryn) [reply tests/dummy/src/resources/user.ts]: Can we not have these kinds of discussions on GH? That is what Slack is for FFS

**Pull request closed without merging**